bundlex.exs

```diff
@@ -25,12 +26,14 @@ defmodule Shmex.BundlexProject do
         sources: ["lib.c"],
         libs: if(Bundlex.platform() == :linux, do: ["rt"], else: [])
       ],
-      lib_nif: [
-        deps: [shmex: :lib, bunch_native: :bunch_nif],
+      lib_iface: [
+        interface: :nif,
+        deps: [shmex: :lib, bunch_native: :bunch_iface],
```

Comment:
Something is off with the naming, once its `lib_iface` then `bunch_iface`.
BTW, how do you choose between `lib_iface` cnode and nif versions?

Reply:
Bundlex chooses the proper version automatically